### DISCUSSION TITLE
XION Chain Onboarding Proposal

### DISCUSSION THREAD

**Discussion by BFondue** [wormhole-foundation/wormhole#3882] [Gateway]
### Website

https://xion.burnt.com

### Github

https://github.com/burnt-labs

### Blog

https://xion.burnt.com/blog

### Documentation

https://docs.burnt.com/xion

### Community stats

- 540,000+ Followers on Twitter 
- 500,000+ Users on Discord
- 135,000+ Followers on Telegram 

### Short Summary

XION is the first L1 blockchain for crypto abstraction. It enables developers to build, launch, and scale consumer-ready Web3 products from the ground up. XION's Generalized Abstraction is an innovative infrastructure solution that aims to abstract away the complexities associated with blockchain technology, making it more accessible for mainstream adoption. By integrating complex functionalities such as accounts, signatures, fees, and interoperability directly at the protocol level, XION eliminates significant barriers to entry for new users and circumvents the fragmentation challenges faced by developers in other ecosystems.

### Why should Wormhole add this chain?

- **Light lift for Guardians** - Guardians are not required to run a full node for XION - connectivity will be via a Gateway IBC connection.
- Other bridges that are already connected to the network: Axelar
- Details and specific metrics about the ecosystem (dApps, users, etc): over 1.5M accounts created on testnet, more than 20M transactions, an ecosystem of over 150 projects, and recently raised 25M.
- Details about how your chain will use Wormhole messaging or token bridging: XION is one of the fastest growing ecosystems and Wormhole will be able to facilitate the flow of assets in and out of the ecosystem.

### Technology and Features

- Generalized Abstraction: all-encompassing infrastructure solution aimed at removing all complexities inherent to crypto to enable mainstream adoption. Its foundation is built upon the seamless integration of complex blockchain functionalities – such as accounts, signatures, fees, and interoperability – directly at the protocol level
- Account abstraction: protocol-level abstracted accounts decoupling the conventional private-public key model. They enable intuitive user experiences traditional in Web2, which includes seamless onboarding, account recovery, modular rule sets, and diverse authentication methods
- Signature abstraction: supports any cryptographic curves to ensure a future proof network, seamless interoperability, while catering to all user audiences
- Parameterized fee layer: a novel approach to fee handling on the network, enabling the abstraction of gas fees from the end-user perspective, the ability for XION to be denominated in USDC, and any token to be accepted for transactions throughout the network

### Native token details

200,000,000 supply at genesis. The XION token is a utility token that serves multiple functions across the layer one network and its ecosystem. These include network usage fees, staking, governance, collateral & liquidity, and medium of exchange.

### Technical requirements for running a node

None - XION will emit messages to Gateway via IBC. Guardians should not have to run anything additional (except for guardians who may run IBC relayers).

### IBC Relaying Strategy

XION will have multiple validators running IBC relayers to Wormhole Gateway. IBC channels are currently already opened with Noble, Osmosis, Axelar, and Injective.

**Comment by Pumckuu** [comment]
мне показалось, что xion полное Г
